### PR TITLE
Add 'question' function

### PR DESCRIPTION
**Pull request by dgendill**
This PR adds the `question` function from the readline API.

### CODE REVIEW

File: src/Node/ReadLine.js [hunk 1]
```diff
@@ -31,6 +31,18 @@ exports.prompt = function (readline) {
...
 };
 
+exports.question = function(text) {
+    return function(callback) {
+        return function(readline) {
+            return function() {
+                readline.question(text, function(result) {
+                    callback(result)();
+                });
+            }
```
Comment (paf31) [inline src/Node/ReadLine.js]: Could you please add trailing semicolons to these `return` statements, to be consistent?

File: src/Node/ReadLine.purs [hunk 2]
```diff
@@ -100,6 +101,16 @@ foreign import prompt
    . Interface
   -> Eff (readline :: READLINE | eff) Unit
 
+-- | Writes a query to the output, waits
+-- | for user input to be provided on input, then invokes
+-- | the callback function
+foreign import question
+  :: forall e eff
```
Comment (paf31) [inline src/Node/ReadLine.purs]: `e` and `eff` should be the same here.

File: src/Node/ReadLine.purs [hunk 3]
```diff
@@ -100,6 +101,16 @@ foreign import prompt
...
+   . String
+  -> (String -> Eff e Unit)
```
Comment (paf31) [inline src/Node/ReadLine.purs]: `e` should be `(readline :: READLINE | eff)`.

**Comment by paf31** [comment]
Thanks!

**Comment by dgendill** [comment]
Thank you!

**Pull request merged**